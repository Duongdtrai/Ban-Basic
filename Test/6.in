7
6
988 175 561 816 56 284 
17
446 443 555 629 339 73 60 789 432 390 523 778 837 456 13 265 45 
12
549 314 712 461 444 146 167 216 899 265 553 104 
37
351 284 949 731 379 256 92 364 178 110 928 590 405 504 895 224 544 457 124 512 386 423 168 684 229 667 41 827 47 760 683 143 621 771 170 792 995 
44
671 242 900 591 923 463 142 133 945 614 682 890 564 41 313 398 587 347 399 839 717 419 633 683 941 125 582 470 208 426 991 143 162 511 608 135 896 850 111 999 996 250 482 547 
36
149 105 535 10 650 274 719 880 889 121 425 738 382 956 658 808 780 117 658 473 932 991 542 990 844 748 461 145 894 794 800 470 691 788 698 632 
33
431 977 400 136 871 103 987 230 955 679 741 795 695 695 150 562 35 708 761 998 529 795 674 413 401 698 199 582 549 764 596 966 192 
