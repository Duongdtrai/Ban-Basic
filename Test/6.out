5441878
225994854
31597108
8869045716
31570928200
3663109848
23098409684
